sphinx/source/docs/user_guide/extensions.rst

```diff
@@ -12,13 +12,13 @@ custom user extensions.
 
 * Modify the behavior of existing Bokeh models
 * Add new models to connect third-party JavaScript libraries to Python
-* Create highly specialized models for domain specific use-cases.
+* Create highly specialized models for domain-specific use-cases.
 
 Custom extensions can be made and used with standard releases, and do not
 require setting up a development environment or building anything from source.
 They provide the easiest way to get involved in Bokeh development. By lowering
 the bar for extending Bokeh, users are afforded the ability to "try out" new
-features and functionality (which might some day be candidates for adding to
+features and functionalities (which might someday be candidates for adding to
```

Comment:
functionalities rings strangely. Perhaps "try out new features and new functionality" ?

Reply:
How about 

'the ability to "try out" new features and improved functionality' 

to not sound too repetitive?

Reply:
sounds good!